src/PageCharacter/CharacterDisplay/Tabs/TabOptimize/Components/OptimizationTargetSelector.tsx

```diff
@@ -28,10 +30,13 @@ export default function OptimizationTargetSelector({ optimizationTarget, setTarg
   const { database } = useContext(DatabaseContext)
   const displayHeader = usePromise(() => optimizationTarget && getDisplayHeader(data, optimizationTarget[0], database), [data, optimizationTarget, database])
 
+  if (!defaultText) defaultText = t("targetSelector.selectOptTarget")
+
   const { title, icon, action } = displayHeader ?? {}
   const node: NodeDisplay | undefined = optimizationTarget && objPathValue(data.getDisplay(), optimizationTarget) as any
 
   const invalidTarget = !optimizationTarget || !displayHeader || !node
+    || (optimizationTarget && data.getDisplay()?.[optimizationTarget[0]][optimizationTarget[1]]?.isEmpty)
```

Comment:
For multi-target, there is a case where a user might get a "invalid" target, say for crescent pike, from share/import config. This invalid value should remain in the list.